packages/api/src/utils/createTask.ts

```diff
@@ -37,6 +42,37 @@ import View = google.cloud.tasks.v2.Task.View
 // Instantiates a client.
 const client = new CloudTasksClient()
 
+/**
+ * we want to prioritized jobs by the expected time to complete
+ * lower number means higher priority
+ * priority 1: jobs that are expected to run immediately
+ * priority 5: jobs that are expected to run in less than 10 seconds
+ * priority 10: jobs that are expected to run in less than 1 minute
+ * priority 50: jobs that are expected to run in less than 30 minutes
+ * priority 100: jobs that are expected to run in less than 1 hour
+ **/
+export const getJobPriority = (jobName: string): number => {
```

Comment:
Ah this is nice. One thing I'd say is we should probably log when we hit default so we handle jobs not having priority set properly.